src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -122,6 +167,10 @@ public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
      * например: "01 January 2000 18:00",
      */
     public String formatDate(LocalDateTime dateTime) {
-        return "";
+        try {
+            return dateTime.format(DATE_TIME_FORMATTER);
+        } catch (DateTimeParseException e) {
+            return "Date can not be formatted!";
```

Comment:
where can you see that this message should be returned?

Reply:
This is not in the description of the task, but I must return some message to the user.
No?

Reply:
Done!